charts/kyverno/templates/validate.yaml

```diff
@@ -8,6 +8,19 @@
   {{- end }}
 {{- end }}
 
+# Function to check replicaCount of all deployments
```

Comment:
this could go in `templates/helpers/_deployment.tpl`

Reply:
Done